include/exadg/acoustic_conservation_equations/spatial_discretization/spatial_operator.h

```diff
@@ -41,6 +41,15 @@ namespace ExaDG
 {
 namespace Acoustics
 {
+/**
+ * Spatial operator to solve for the acoustic conservation equations:
+ *
+ * 1/c^2 * dp/dt + rho* div  u = f
+ *   rho * du/dt +      grad p = 0
+ *
+ * This operator solves for the pressure p as well as the velocity that is
+ * already scaled by the density rho * u.
```

Comment:
Given this description, I would in the equations above either skip rho, or write 
- `div (rho * u)` in the first equation, and 
- `d (rho * u)/dt` in the second equation.